docs/applications/configuration-management/configure-and-use-salt-ssh.md

```diff
@@ -0,0 +1,216 @@
+---
+author:
+  name: Sergey Bulavintsev
+  email: bulavintsev.sergey@gmail.com
+description: 'Salt SSH will allow you to manage your minions without having to install salt-minion agent. Learn how to configure and use Salt SSH in this simple tutorial'
+keywords: 'Saltstack, salt, salt-ssh, roster'
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 'Wednesday, July 25, 2017'
+modified: 'Tuesday, August 15th, 2017'
+modified_by:
+  name: Linode
+title: 'Configuring and Using Salt SSH to Manage Your Linodes'
+contributor:
+  name: Sergey Bulavintsev
+---
+
+#Introduction
+Salt-ssh allows you to execute Salt commands or states without installing a salt-minion package.
+This allows Salt to work similarly to Ansible, where master is pushing updates to the minions.
+During execution, salt-ssh will copy necessary files to the target system via ssh to the `/tmp` folder, execute command or state and clean up salt temporary files.
+Be aware that Salt SSH is slower than standard Salt with ZeroMQ because it it working via SSH.
+
+
+#Before You Begin
+
+1.  This guide assumes that you're using rpm based (CentOS, RedHat, Oracle Enterprise Linux) system.
+
+2.  Make sure that you have salt and salt-ssh packages installed on your master. You may check if these packages are installed by executing:
+
+        $rpm -q salt
+        $rpm -q salt-ssh
+
+    {: .note}
+    >
+    > For detailed instruction on how to set up SaltStack repo, please refer to [SaltStack installation](https://www.linode.com/docs/applications/configuration-management/install-and-configure-salt-master-and-minion-servers)
+
+3.  Your minions must have Python installed. Without Python installed on minions, you will only be able to run salt-ssh in raw mode. If you're running any modern version of CentOS/RedHat, you should already have Python installed on your systems. In raw mode, you execute a raw shell command and cannot use execution modules or apply salt states.
+
+4.  You must have at least one master server and one minion (client).
+
+##Set up Salt Roster file
+
+Roster file contains information on target systems, connection details and credentials, which will be used for connection.
+Default location for roster file is `/etc/salt/roster`.
+
+   {: .note}
+   >
+   > Roster file is configured on master server.
+
+1.  Open the file `/etc/salt/roster` with your editor of choice and add following lines to define client systems:
+
+    This is an example of minimal host definition
+
+    {: .file }
+    /etc/salt/roster
+    :  ~~~ config
+    linode1:
```

Comment:
You receive this error because I assumed: " If you already have your public key installed on minion and you have private key on master".
If SSH keys weren't deployed yet, you have 2 options:
1. Connect to your minion via SSH and manually accept it's public key
2. Execute salt-ssh with "-i" key. This key will let Salt automatically accept minion's public key. This has to be done only once, only during initial SSH keys exchange.